### ISSUE TITLE
No proper road map

### ISSUE THREAD

**Issue by pl23** [Alexofp/BDCC#114]
It's hard to tell what needs to be worked on because there's no public road map as somebody who wants to contribute it's hard to tell what needs to be done it almost seems like we could just create anything when it should just be a mod

**Comment by pl23** [comment]
This is a follow-up to my earlier post I know there's an easy way to add (projects) as GitHub likes to call them which would allow you to make a timeline or road map it's just confusing to see what's in development and what isn't

**Comment by CanInBad** [comment]
pay for subscribestar for as low as 1 usd #ad #promoted #pleasegivethebreadcatmoney

**Comment by pl23** [comment]
Just so I'm understanding you correctly we should pay to contribute our time and energy to develop this further I'm not talking about story beats to spoil any future plot development but to add more features and game functionality? Also it's very confusing to tell who's in charge of what?

**Comment by pl23** [comment]
I understand it's only a single US dollar but I personally don't understand when there could just be a road map in the GitHub repository itself

**Comment by pl23** [comment]
And from the brief look at the site you suggested it doesn't look like there's a roadmap there might be one in there and I might need to do more digging but I did not find one

**Comment by pl23** [comment]
The basic question is what is the future plans of development if any

**Comment by pl23** [comment]
I leave this issue open for anyone to comment

**Comment by CanInBad** [comment]
its an open source project lol, you can add anything you want (see #121 #111 #41 ), only issue is if the pr has some problems with the repo owner

there might be some hint of what is happening inside 
 commit messages, but if you want to know more what changes planned in next 3 update then yeah you have to pay to know.

also I see that you didn't fork this repo so what feature are you even working on?
you mentioned that you want to contribute but yet there is no trace of such in the work.

you don't really get "roadmap" persay, its more like a  devlog inside our discord server, the "roadmap" will be updated/reposted every new releases.

**Comment by pl23** [comment]
Saying I'm working on anything I'm just sitting on the sidelines I'm giving my opinion I'm just saying as an aspiring developer it's kind of hard to tell what needs to be worked on for example is on a road map that I was kind of looking for I recommend looking at the Star citizen road map although that's a bit of an extreme case just something that lets other people know what needs to be worked on would help I'm not saying I'm trying to add my own features I understand it's an open source project in anyone can add anything but for long-term goals is kind of what I was looking for

**Comment by pl23** [comment]
I don't know just something to show more long-term commitment what are the future plans I understand that we plan on adding another story branches. But that's not what I'm referring to I was thinking more on the functional side if we needed to rework the inventory system or if we wanted to add another feature to the game etc etc possibly future thoughts for DLC perhaps?

**Comment by pl23** [comment]
I don't know just some discussion on future plans maybe the owner doesn't even have any plans I don't know I'm not him or she

**Comment by pl23** [comment]
And finally I don't make public forks I hardly fork unless I see their needing to be a reason to

**Comment by pl23** [comment]
And when I do I just clone the whole repo for me to work on separately on my own time

**Comment by AverageAcee** [comment]
I get not wanting to waste time on developing a feature that is already being worked on but the game is open source already.
You can quite clearly see what is being worked on via github commits and failing that there are places that could always use more content like story or character customization.
I would familiarize yourself with the community and the game rather than naively demand a AAA style live service roadmap for an indie game with a single full time dev where new content is devlogged and hinted at. Unless you just go the devbuild of the game freely available to anyone on the game's discord and spoil the new stuff yourself.
Also having to be blatantly told what needs to be worked on is quite a misunderstanding of how things are run and worked on.
I personally have made around 90 mods for the game and all of them have come from the idea of "oh hey I should add this into the game" and just going ahead and doing it.
Things are far more casual.

And the talk of future DLC is just confusing. That is so far away from where we are now and not at all required.

**Comment by pl23** [comment]
That might be a bit out of scope but I like the direction however I feel it's not our place to decide characters and story and well the discord does have a suggestion box

**Comment by pl23** [comment]
I was referring to limitations like what we would need to do in order to add more features like more mini games etc etc but the fact there's no technical detail makes it difficult like as a graphics artist I would like to work on some of the graphics but I don't know what needs to be worked on or if it needs to be worked on

**Comment by pl23** [comment]
Should we consider the game engine to be complete?

**Comment by pl23** [comment]
Like I understand trying crowdfund for further development but because it's open source it makes it a little difficult to understand what needs to be made or fixed

**Comment by pl23** [comment]
I only used it an example just some general knowledge on what needs to be completed would help

**Comment by pl23** [comment]
Just something to know who's responsible for working on what. And everyone would still needs to be working. Should we consider the game engine complete? if so then what? Do we just need to wait on more of the story to be developed? A little clear to you would be appreciated

**Comment by pl23** [comment]
I'm not trying to make this a a place where we can have future creep but knowing what to do is the battle

**Comment by pl23** [comment]
They do have a discord suggestion box for story related stuff or anything really but when it comes to people actively working on it I believe knowing what needs to be done here needs to be more fleshed out

**Comment by pl23** [comment]
Look I'll even take a to-do list posted inside of the readme

**Comment by AverageAcee** [comment]
I genuinely have no idea what you're trying to accomplish.
I have told you to familiarize yourself with the community and source code of the game in order to find something yourself rather than relying on a static list. Maybe you can find a feature you want you can add into the game? The hypnotist fetish was made by one guy. If you are truly that determined you can do something similar with a mod. Hell if you want you can take on the endless list of species mods. The game's discord may have problems but it's the best place to get ideas on how and what to modify the game.
it can be boiled down to a simple question
What is something you have wanted in the game? 
and then once you have your answer, go ahead and start making it.

**Comment by pl23** [comment]
Yeah  that's fair enough
